apps/editor.planx.uk/src/pages/FlowEditor/lib/store/shared.ts

```diff
@@ -117,14 +115,6 @@ export const sharedStore: StateCreator<
     ).has(id);
   },
 
-  setFlowSlug(flowSlug) {
-    set({ flowSlug });
-  },
-
-  setFlowName(flowName) {
-    set({ flowName });
-  },
```

Comment:
I've dropped these are they were only being used in one place and don't offer much more than a plain `set()` does. Eventually, these values can just be held in the Apollo cache (or route data) as opposed to within the store.